src/scss/config/scale/_theme-sizes.scss

```
@@ -1,4 +1,11 @@
+@use 'sass:math';
+@use 'page-sizes';
+
 $page-margin: calc(var(--quarter-shim) + 4vw);
 $border-width: medium;
 $radius: var(--half-shim);
 $grid-hero: calc(var(--gutter) * 8);
+$logo-item: 10rem;
+$grid-item: 16rem;
+$main-min: calc(100vw - var(--page-margin) * 2);
+$extra: calc(var(--wide-page) - var(--page) / 2);
```

```suggestion
$extra: calc((var(--wide-page) - var(--page)) / 2);
```

Otherwise the division happens first

---

good catch. fixed in https://github.com/oddbird/oddleventy/pull/306/commits/088d051b4dd722f80ad7747508b815562102895c